README.md

```diff
@@ -27,6 +27,7 @@ All contributors to any Work Items must be members of the Solid CG. It’s easy
 Anyone can join the [Authorization Panel chat](https://gitter.im/solid/authorization-panel).
 
 Online panel meetings occur sporadically on Wednesday, 14:00 UTC at https://meet.jit.si/solid-authorization.
+They are announced and can be requested on the Authorization Panel chat.
```

Comment:
Live/instant-chat-space is a terrible calendaring mechanism, not least because (as has been expressed on recent minutes) keeping track of all the various conversation spaces (Gitter, Twitter, IRC, GitHub, Slack, email, etc.) is difficult if not impossible.

Meetings should be announced via a calendaring tool such as the W3 Calendar.

Requests would probably be better via a more typically asynchronous tool, such as email or github issue, with associated polling mechanism, to help make chosen timing more convenient for larger numbers of potential attendees.

Reply:
Ok, I retract this suggestion.